foxtrot-core/pom.xml

```diff
@@ -80,7 +79,7 @@
         <dependency>
             <groupId>com.flipkart.foxtrot</groupId>
             <artifactId>foxtrot-common</artifactId>
-            <version>0.3</version>
+            <version>${project.parent.version}</version>
```

Comment:
Good .. Can you also move all dependency versions to properties? Can be a separate pull request.